test/BUILD

```diff
@@ -30,6 +30,8 @@ ts_library(
 jasmine_node_test(
     name = "unit_test",
     srcs = [":unit_test_lib"],
+    # Unit tests need access to tslib.
```

Comment:
shouldn't that rather be a "deps" entry on unit_test_lib then? data is not how TS dependencies are supposed to work.

Reply:
The tests run the TS compiler, which needs tslib.d.ts as an input data file.  I clarified the comment.

The way this works is that if you have importHelpers on, and your code does e.g.
  x = {...y}
then TS wants to use its tslib shim for Object.assign().  To do so it attempts to module resolve 'tslib'.  If tslib isn't found (or if it doesn't have a __assign export) then it's a compile error.

So for the tests to be able properly run TypeScript compilation, they need to ensure that the compiler they run succeeds in resolving tslib.